domain/robstoll-lib/atrium-domain-robstoll-lib-common/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/throwableAssertions.kt

```diff
@@ -11,10 +11,10 @@ import ch.tutteli.atrium.translations.DescriptionThrowableAssertion
 import kotlin.reflect.KClass
 
 @UseExperimental(ExperimentalWithOptions::class)
-inline fun <TExpected : Throwable> _cause(
-    expect: Expect<Throwable>,
+inline fun <T: Throwable, TExpected : Throwable> _cause(
```

Comment:
remove inline here (btw. you can also commit suggestions from the GitHub website)
```suggestion
fun <T: Throwable, TExpected : Throwable> _cause(
```